app/views/admin/ambassadors/show.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1= "Ambassador: #{@ambassador.username}"
+
+- if @ambassador.avatar
+  - unless @ambassador.avatar.url(:medium) == 'https://files.bikeindex.org/blank.png'
+    = image_tag @ambassador.avatar.url(:medium), class: 'users-show-avatar'
+
+
+%table.table.table-striped.table-hover.table-bordered.table-sm.without-exterior-border
+  %tbody
+    - @ambassador.attributes.slice(*@user_attributes).each_pair do |key, value|
```

Comment:
I think I would prefer this list to be defined here rather than the controller.

Reply:
I still feel the same way about this - I feel like this is a view concern, and would prefer to define the `USER_ATTRIBUTES` here, rather than in the controller.

But this is a nit, and isn't actually important, and this is better than nothing!

I think in the future, we should create a partial for admin user display, like we have for bike display, and include that partial in the admin user edit, here and any other relevant admin places.

Reply:
Ah, I misread your last (slightly ambiguous antecedent to "this", my bad). 
Added a decorator in df9185c7

Reply:
Yeah, sorry, I definitely didn't explain myself very well 🙂